### PR TITLE
[Feature] support gpt-3.5-turbo-1106

### PR DESCRIPTION
**Pull request by zjc17**
image

### CODE REVIEW

**Comment by zjc17** [comment]
@songquanpeng Appreciate to your work and this commit adds support for the latest released model

File: common/model-ratio.go [hunk 1]
```diff
@@ -99,5 +100,8 @@ func GetCompletionRatio(name string) float64 {
 	if strings.HasPrefix(name, "claude-2") {
 		return 2.965517
 	}
+	if strings.HasPrefix(name, "gpt-3.5-turbo-1106") {
```
Comment (waltcow) [inline common/model-ratio.go:103]: if strings.HasPrefix(name, "gpt-3.5") 
这个已经match了，就不会到这里了

**Comment by songquanpeng** [comment]
Thanks for your PR, but a duplicated one (#696) has been merged.

**Comment by songquanpeng** [comment]
Close for now

**Pull request closed without merging**